unitOneAssessmentPractice.js

```diff
@@ -4,34 +4,74 @@ let assert = require('assert')
 
 // Write a function called double that doubles a number
 
+const double = (num) => {
+  return num * 2
+}
+
 // Uncomment out the next line to test your solution
-// runQ1Tests()
+runQ1Tests()
 
 // Question Two:
 
 // Write a function called containsSeven that returns whether or not a number has an sevens in it.
 
+const containsSeven = (num) => {
+    return num.toString().includes("7")
+} 
+
 // Uncomment out the next line to test your solution
-// runQ2Tests()
+runQ2Tests()
 
 // Question Three:
 
 // Write a function called capitalizeTheAs the capitalizes all of the lowercase "a"s in a string leaving all other characters the same
 
+const capitalizeTheAs = (string) => {
+  let upperCase = "";
+  for(let i = 0; i < string.length; i++) {
+    if(string[i] === "a") {
+      upperCase += string[i].toUpperCase()
+    } else {
+      upperCase += string[i]
+    }
+  }
+  return upperCase
+}
+
 // Uncomment out the next line to test your solution
-// runQ3Tests()
+runQ3Tests()
 
 // Question Four:
 // Write a function called largest that returns the largest value in an array
 
+const largest = (array) => {
+  let value = 0
```

Comment:
This looks good, but in the case it has negative numbers your value can start at `-Infinity` since all the values in the array can be negative and this would give you the wrong output